app/markdown/governing-docs/constitution.md

```diff
@@ -290,10 +295,21 @@ Vice President  | 3             | 1 academic term
 Treasurer       | -             | 1 academic term
 Secretary       | -             | - 
 
-“Year level” is counted by the lower of [A] years enrolled at least part-time in a higher-education institution post high school graduation, or [B] by RIT credits obtained following the scale published by the RIT Office of the Registrar.  “Prior Involvement” refers to the number of complete academic terms during which the candidate served as a Primary Officer or Committee Head. In the event a candidate has not previously served as a Primary Officer or Committee Head, two complete academic terms during which the candidate served as a Mentor with the society may be substituted to serve as one term of “Prior Involvement”. No more than one term of “Prior Involvement” earned through mentorship may be substituted as the requirements for any office.
+“Year level” is counted by the lower of [A] years enrolled at least part-time in a
+higher-education institution post high school graduation, or [B] by credits obtained
+at the Rochester Institute of Technology following the scale published by the Rochester
+Institute of Technology's Office of the Registrar.
```

Comment:
My changes here just for formatting
I think [B] is confusing, personally. Is it meant to count year level based on how many credits earned by taking classes at RIT?